lib/less/tree/import.js

```diff
@@ -127,7 +127,7 @@ Import.prototype.eval = function (context) {
         }
     }
 
-    if (this.options.inline) {
+    if (this.options.inline && !this.path.currentFileInfo.reference) {
```

Comment:
Just to be sure, is this about the difference in the output of the the following (artificial) example:
```
// ref.less
.mixin() {
    @import (inline) "boo.css"
}
```
```
// master.less
@import (reference) "ref";
.mixin();
```
?